File: Iserv.Niis.Model/Models/ConvertDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Iserv.Niis.Domain.Helpers;

namespace Iserv.Niis.Model.Models
{
    public class ConvertDto
    {
        public int OwnerId { get; set; }
        public Owner.Type OwnerType { get; set; }
        public string ColectiveTrademarkParticipantsInfo { get; set; }
    }

    public class ConvertResponseDto
    {
        public string ColectiveTrademarkParticipantsInfo { get; set; }
        public int? SpeciesTradeMarkId { get; set; }
        public string SpeciesTrademarkCode { get; set; }
    }
}

File: Iserv.Niis.Angular/src/app/bibliographic-data/models/convert-dto.ts
import { OwnerType } from 'app/shared/services/models/owner-type.enum';

export class ConvertDto {
  ownerId: number;
  ownerType: OwnerType;
  colectiveTrademarkParticipantsInfo: string;
}

export class ConvertResponseDto {
  colectiveTrademarkParticipantsInfo: string;
  speciesTradeMarkId: number;
  speciesTrademarkCode: string;
}
